parser/start_tokens.coffee

```diff
@@ -12,7 +12,7 @@ class StartTokens
   @expression = [
     'INTLIT', 'FLOATLIT', 'true', 'false', # Primitive literals
     'ID', '->', 'trait', 'class',          # Misc other tokens
-    @iterable, tokens.reservedWords,
+    @iterable, tokens.beginnings,
```

Comment:
If you merge master into this branch and update the handling of start tokens, this branch should be good to go!